### ISSUE TITLE
Recaptacha V3

### ISSUE THREAD

**Issue by jimblue** [aryehraber/statamic-recaptcha#5]
Hey!

Just to let you know recaptacha v3 is out with a lot of new features:

https://webmasters.googleblog.com/2018/10/introducing-recaptcha-v3-new-way-to.html

It could be nice to update **statamic-recaptcha** to use this last release 😄 !

Cheers

**Comment by aryehraber** [comment]
Hey @jimblue, thanks for the heads up! Will try and take a look at this when I next have a moment.

**Comment by edalzell** [comment]
FYI, the back end works fine with v3, once you update the front end stuff.

@aryehraber this would be amazing, let me know if you'd like to see how I got it working.

**Comment by aryehraber** [comment]
@edalzell Noice, please share 😊

**Comment by twd3** [comment]
Any news on this? I have the V2 reCAPTCHA working on a V3 site but it would be great to have V3! Also, the V3 has an option where the keys will work for AMP pages so making this work would help solve #15 ! Thanks again for updating this V2 for Statamic V3!

**Comment by aryehraber** [comment]
Hey @twd3, no news at the moment. 

I had a brief chat with @edalzell on Twitter ([thread here](https://twitter.com/emd/status/1251559574940921859)) about this a few months ago but still don't really see the upside of V3 and am waiting for a compelling argument (and proposed solution) to support it. 

Basically, what is it about Recaptcha V3 that you are waiting for that V2 can't already do? 

In my research so far it seems like it's a worse experience on both sides (users and admins) for regular websites, depending on implementation: 

1) The user's submission is blocked because they were marked "risky" by Google and their score is below the allowed threshold. This would mean they literally can't submit the form, or it silently fails and submissions never arrive in an admin's inbox.

2) We store all "risky" submissions in a spam folder and stop auto-emails from being triggered, but now the burden is put on the admin to go and check spam everyday and mark Google's incorrect "risky" submissions as not spam (triggering auto-emails, etc). Also worth noting that Statamic doesn't natively support a spam folder so this solution would be a much larger undertaking since that needs to be built and integrated, instead of the current solution which simply makes use of existing core features.

I can see V3's usefulness in larger applications where the score can lead users to an additional security flow before being allowed to take a certain action, but I just don't see that being a good solution for normal forms on a regular website.

To reiterate, I'm completely open to the idea but am currently not convinced about it in a Statamic context. For more complex Statamic projects it could be useful, but at that stage you probably want to control the actions that are taken for low/high scores meaning a generic solution like this addon wouldn't really be possible/useful.

Please share your ideas and vision for use cases, I'll happily change my mind!

**Comment by twd3** [comment]
Well, the only thing is I really don't think V2 is working anymore. I think the bots figured it out. I've had the V2 (for Statamic V3) installed and live for a week now and the spam is coming through like crazy. On Wednesday I turned the "security" up all the way. Just last night I got 0 leads and 20 spams. It is so bad I cannot let the executives get the email notifications right now because it would make me look so bad as their developer so I have to look for actual leads right now and then forward them out. It's bad because they really need to respond to these leads right away and this makes a lag.

I guess I don't understand why honeypot and this V2 solution doesn't work for me.
